core/app/models/spree/app_configuration.rb

```diff
@@ -74,14 +74,12 @@ class AppConfiguration < Preferences::Configuration
     # @return [Boolean] automatically capture the credit card (as opposed to just authorize and capture later) (default: +false+)
     preference :auto_capture_exchanges, :boolean, default: false
 
-    # @!attribute [rw] binary_inventory_cache
-    #   Only invalidate product caches when they change from in stock to out of
-    #   stock. By default, caches are invalidated on any change of inventory
-    #   quantity. Setting this to true should make operations on inventory
-    #   faster.
-    #   (default: +false+)
-    #   @return [Boolean]
-    preference :binary_inventory_cache, :boolean, default: false
```

Comment:
For the time being, I'd prefer if we kept both of these in place. We shouldn't remove this without first deprecating it.